xbmc/windowing/osx/WinSystemOSX.h

```diff
@@ -59,6 +59,17 @@ class CWinSystemOSX : public CWinSystemBase, public ITimerCallback
 
   std::string GetClipboardText() override;
 
+  /*! \brief Check if the windowing system supports moving windows across screens
+   *  \return true if the windowing system supports moving windows across screens, false otherwise
+  */
```

Comment:
not sure that copy-pasting docstring from parent class is needed

Reply:
doesn't harm either :)